File: app/src/main/java/com/cdh/okonedemo/DemoAPP.java
package com.cdh.okonedemo;

import android.app.Application;

import com.cdh.okone.OkOne;

/**
 * Created by chidehang on 2020/12/12
 */
public class DemoAPP extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        OkOne.setLogEnable(true);
    }
}

File: app/src/main/java/com/cdh/okonedemo/DemoAPP.kt
package com.cdh.okonedemo

import android.app.Application
import com.cdh.okone.OkOne

/**
 * Created by chidehang on 2020/12/12
 */
class DemoAPP : Application() {
    override fun onCreate() {
        super.onCreate()
        // 配置OkOne
        OkOneConfigur.config()
    }
}